extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -250,28 +250,28 @@ export class Bind {
 
   /**
    * Merges `state` into the current state and immediately triggers an
-   * evaluation unless `opt_skipEval` is false.
+   * evaluation unless `skipEval` is false.
    * @param {!JsonObject} state
-   * @param {boolean=} opt_skipEval
-   * @param {boolean=} opt_skipAmpState
+   *
+   * @param {BindSetStateOptionsDef} opts options bag
```

Comment:
Nit: Prefer explicit nullability i.e. `!BindSetStateOptionsDef`. https://google.github.io/styleguide/jsguide.html#jsdoc-nullability

Reply:
Thanks for the pointer. Done